cookietemple/cookietemple_cli.py

```diff
@@ -212,10 +212,11 @@ def sync(project_dir, set_token, pat, username, check_update) -> None:
             log.debug('Starting sync.')
             syncer.sync()
         else:
-            print('[bold red]Aborting sync due to set level constraints. You can set the level any time in your cookietemple.cfg in the sync_level section and'
-                  ' sync again.')
+            console.print(
```

Comment:
This is really ugly intendation with the brackets.
Almost looks like Black formatting haha.

Reply:
Please go for
console.print (1sentence
                       2 sentence)

Reply:
See below